src/plugins/kibana_react/public/page/page_footer/page_footer.tsx

```diff
@@ -0,0 +1,115 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { FC } from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiButtonEmpty } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+import { CoreStart } from 'kibana/public';
+import { RedirectAppLinks } from '../../app_links';
+import { useKibana } from '../../context';
+import { useUiSetting$ } from '../../ui_settings';
+
+import './index.scss';
+
+interface Props {
+  /** The path to set as the new default route in advanced settings */
+  path: string;
+}
+
+export const PageFooter: FC<Props> = ({ path }) => {
+  const [defaultRoute, setDefaultRoute] = useUiSetting$<string>('defaultRoute');
+  const {
+    services: {
+      application,
+      notifications: { toasts },
+    },
+  } = useKibana<CoreStart>();
+
+  const { show, save } = application.capabilities.advancedSettings;
+  const isAdvancedSettingsEnabled = show && save;
+
+  const defaultRoutebutton =
+    defaultRoute === path ? (
+      <RedirectAppLinks application={application}>
+        <EuiButtonEmpty
+          className="pageFooter__button"
```

Comment:
All the `pageFooter` classes need the `kbn` prefix and ideally would be more specific even though the name of the component is simply `PageFooter` because it's so general it could conflict in the future.

Reply:
I changed the prefixes to `kbnOverviewPageFooter` and `kbnOverviewPageHeader`